hera_pspec/pspecdata.py

```diff
@@ -12,14 +12,18 @@
 import warnings
 import json
 import uvtools.dspec as dspec
+# This is a list of weights that will depend on baseline.
```

Comment:
give two line breaks after import statements

Reply:
I've moved this.